src/test/prettyPrinter.spec.ts

```diff
@@ -1,5 +1,5 @@
 import fs from "fs";
-import { prettyPrint } from "../prettyPrinter";
+import { prettyPrint } from "../ast/ast-printer";
```

Comment:
Could you please also move `src/test/prettyPrinter.spec.ts` here? It's somewhat odd it's not next to it.